man/systemd.network.xml

```diff
@@ -317,6 +317,47 @@
       </variablelist>
   </refsect1>
 
+  <refsect1>
+      <title>[MPTCP] Section Options</title>
+      <para>The [MPTCP] section accepts the following keys. Specify several [MPTCP] sections to configure
+      several MPTCPs. MPTCP is a transport protocol built on top of TCP that allows TCP connections to use
+      multiple paths to maximize resource usage and increase redundancy.</para>
+
+      <variablelist class='network-directives'>
+        <varlistentry>
+          <term><varname>Id=</varname></term>
+          <listitem>
+            <para>Specifies a a unique numeric identifier for the given endpoint. Takes an unsigned integer in the range 0..255.</para>
```

Comment:
This setting seems mandatory. If so, please mention so.

Reply:
No it's not required. If not provided kernel automatically increases

Reply:
OK. Then, please introduce `id_is_set` boolean and send the ID only when it is explicitly configured.

BTW, I may misunderstand the kernel code, but is it really used? The ID is parsed here:
https://github.com/torvalds/linux/blob/805c6d3c19210c90c109107d189744e960eae025/net/mptcp/pm_netlink.c#L479-L480
But soon later it will be overwritten:
https://github.com/torvalds/linux/blob/805c6d3c19210c90c109107d189744e960eae025/net/mptcp/pm_netlink.c#L311

Reply:
No you are right. It's a BZ in kernel I think. I put that ID, it does not work.

Reply:
Haha, thanks.